import_craft.py

```diff
@@ -0,0 +1,291 @@
+
+import os
+import bpy
+import import_mu
+from cfgnode import ConfigNode
+
+
+def print_blender_data_stats(label):
+    nbobj = len(bpy.data.objects)
+    nbmat = len(bpy.data.materials)
+    nbtex = len(bpy.data.textures)
+    nbimg = len(bpy.data.images)
+
+    print ('Loading stats ({}) :\
+            \n - {} objects \n - {} materials \
+            \n - {} textures \n - {} images \
+            \n'.format(label, nbobj, nbmat, nbtex, nbimg))
+
+
+def rename_textures(ob):
+    index = 0
+    for tex in bpy.data.textures:
```

Comment:
can't remember why I made the comment so it's probably obsolete; just rewrite more a bit pythonically as we discussed